tailwind.config.ts

```diff
@@ -1,9 +1,11 @@
-//@ts-check
+// @ts-check
```

Comment:
Do we still need this now that it's a ts file?

Reply:
nope - removed